server/src/test/java/io/javaclasses/filehub/api/PasswordValidatorTest.java

```
@@ -0,0 +1,65 @@
+package io.javaclasses.filehub.api;
+
+import com.google.common.testing.NullPointerTester;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static java.lang.String.format;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@DisplayName("Login Validator should ")
+public class PasswordValidatorTest {
+
+    private static Stream<Arguments> invalidPasswords() {
+
+        return Stream.of(
+            Arguments.of("1234567", "has less size"),
+            Arguments.of("qwerty123A#", "contains restricted characters"),
+            Arguments.of("qwerty1234", "doesn't have upper-case Latin."),
+            Arguments.of("QWERTY1234", "doesn't have lower-case Latin."),
+            Arguments.of("qwertyQWERTY", "doesn't have any digit.")
+        );
+    }
+
+    @DisplayName("successfully validate correct input.")
+    @Test
+    void testSuccessfulValidation() {
+
+        Validator validator = new PasswordValidator();
+
+        assertDoesNotThrow(
+            () -> validator.validate("qwerty123A"),
+            "Cannot validate correct data."
+        );
+    }
+
+    @DisplayName("fail validation of invalid input.")
```

I think display name should be: `throw an exception when input is invalid`.